atest/testdata/misc/suites/subsuites2/__init__.robot

```diff
@@ -0,0 +1,2 @@
+*** Settings ***
+Name    Parent Suite Name
```

Comment:
1. It would probably be a good idea to include the word "Custom" in all suite names to make it explicit they are custom names. You used that style with most other suite names.
2. Add newline at the end of the file.

These comments apply also to other suites.

Reply:
changed name to `Custom parent Suite Name`
+ new EOL